3-things-you-must-know-about-midnight-hacks.md

```diff
@@ -0,0 +1,43 @@
+{  
+  "author": "Kara Dees",
+  "title": "3 Things You Must Know About Midnight Hacks",
+  "thumbnail": "http://imgur.com/a/YATmL",
```

Comment:
So I will start using Imgur for thumbnails, but the logic has not been implemented yet, so for now, please upload this picture to the `/images` folder with the name `midnight-hacks.jpg` and change this line for `"thumbnail": "midnight-hacks.jpg"`